VideoLocker/res/layout/fragment_course_combined_info.xml

```diff
@@ -1,86 +1,73 @@
 <ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
-            xmlns:app="http://schemas.android.com/apk/res-auto"
-            xmlns:tools="http://schemas.android.com/tools"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:background="@color/edx_grayscale_neutral_white"
-            android:fillViewport="true"
-            tools:context="org.edx.mobile.view.CourseCombinedInfoFragment">
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/edx_grayscale_neutral_white"
+    android:fillViewport="true"
+    tools:context="org.edx.mobile.view.CourseCombinedInfoFragment">
 
     <LinearLayout
+        android:id="@+id/announcement_layout"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="match_parent"
```

Comment:
Ready for re-review.

On Mon, May 9, 2016 at 11:12 AM, Brian Guertin <notifications@github.com>
wrote:

> In VideoLocker/res/layout/fragment_course_combined_info.xml
> <https://github.com/edx/edx-app-android/pull/690#discussion_r62515969>:
>
> >          android:layout_width="match_parent"
> > -        android:layout_height="wrap_content"
> > +        android:layout_height="match_parent"
>
> The child should be wrap_content. Android studio warns about this, you
> should see it when looking at the file.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-app-android/pull/690/files/5dfda19d45d8c5889a2972d08367056e5d241508#r62515969>
>



-- 
Regards,

Marc DiNino
Android Engineer
EDX